package.json

```diff
@@ -62,28 +66,30 @@
     "postinstall:project": "link-parent-bin && npm run compile && npm run bootstrap",
     "publish": "npm whoami && check-installed-dependencies && npm run compile && npm test && npm run props-table && lerna publish",
     "postpublish": "npm run dependency-markdown && git add --all && git commit -m 'Updated DEPENDENCIES.md files'",
-    "start": "cd packages/terra-framework-site && npm run start",
+    "start": "node node_modules/terra-site/scripts/start-terra-site/start",
```

Comment:
@mjhenkes @bjankord This would spin up terra-site. I think it would be good if we could get this moved to be a bin script such that start-terra-site would be symlinked by any repo using it.